pym/bob/scm/url.py

```diff
@@ -98,24 +98,45 @@ def asScript(self):
 cp -uT {URL} {FILE}
 """
         else:
+            tool = "curl"
+            options = ""
+            useTimeStamping = True
+            if which(tool) is not None:
```

Comment:
The whole descision must be done during runtime in the bash script. The python code runs where the recipes are parsed which is not necessarily the same machine as where they are executed in case of the Jenkins backend.

Reply:
The backend stuff was totally out of scope to me, sorry.  I will update asap.